solana/trie-geyser-plugin/config.json

```diff
@@ -0,0 +1,6 @@
+{
+	"libpath": "target/release/libwitnessed_trie_geyser_plugin.so",
```

Comment:
```suggestion
	"libpath": "target/release/libwitnessed_trie_geyser_plugin.dylib",
```

Reply:
Looks like Mac thing? Looks like there’s no one thing that would work for Linux and Mac. I’ll update the README.

Reply:
oh i see, maybe.